server/app/services/applicant/ReadOnlyApplicantProgramService.java

```diff
@@ -115,6 +114,13 @@ public interface ReadOnlyApplicantProgramService {
    */
   ImmutableList<AnswerData> getSummaryDataOnlyActive();
 
+  /**
+   * Returns summary data for each question in the hidden blocks in this application. Hidden block
+   * is the block not visible to the applicant based on the visibility setting by the admin. This
+   * will not include blocks that are active.
+   */
+  ImmutableList<AnswerData> getSummaryDataOnlyHidden();
```

Comment:
This is unchanged, I just changed the order to match the order in the Impl class